packages/legend-art/src/panel/PanelDropZone.tsx

```diff
@@ -30,10 +30,16 @@ export const PanelDropZone: React.FC<{
   children: React.ReactNode;
   isDragOver: boolean;
   dropTargetConnector: ConnectDropTarget;
+  draggableActive?: boolean | undefined;
```

Comment:
```suggestion
  showDropableSuggestion?: boolean | undefined;
```

Reply:
We should think about the interaction between this and the overlay, I think the border here should be on a layer that is beneath the overlay and above the content, also, it should be absolutely position to not jank the layout